quantgov/estimator/estimation.py

```diff
@@ -61,7 +61,7 @@ def estimate_probability(vectorizer, model, streamer, precision):
     pipeline = get_pipeline(vectorizer, model)
     texts = (doc.text for doc in streamer)
     truecol = list(int(i) for i in model.model.classes_).index(1)
-    predicted = ((round(i[truecol], int(precision))
+    predicted = ((round(i[truecol], precision)
```

Comment:
Why aren't we selecting the whole column and rounding at once? I think that would be `pipeline.predict_proba[:,truecol].round(precision)`.